config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -183,7 +183,13 @@ spec:
                 description: Additional labels to apply to the service
                 type: string
               annotations:
-                description: annotations for the pods
+                description: Common annotations for both deployments
+                type: string
+              task_annotations:
+                description: Annotations specific to the task deployment
```

Comment:
is this an override of `annotations` field or is it a combination with the `annotation`

Reply:
from the code this looks like a override but the description here sounds like it will be additive rather than an override

Reply:
Maybe rewording it to say:
```
                description: Task deployment annotations. This will override the general annotations parameter for the Task deployment.
```